spring-cloud-azure-feature-management/src/main/java/com/microsoft/azure/spring/cloud/feature/manager/FeatureManager.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.feature.manager;
+
+import java.util.LinkedHashMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.spring.cloud.feature.manager.entities.Feature;
+import com.microsoft.azure.spring.cloud.feature.manager.entities.FeatureFilterEvaluationContext;
+import com.microsoft.azure.spring.cloud.feature.manager.entities.FeatureSet;
+
+/**
+ * Holds information on Feature Management properties and can check if a given feature is
+ * enabled.
+ */
+@Configuration
+@ConfigurationProperties(prefix = "feature-management")
+public class FeatureManager {
+
+    Logger logger = LoggerFactory.getLogger(FeatureManager.class);
+
+    FeatureSet featureManagement;
+
+    FeatureSet featureSet;
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Autowired
+    private ApplicationContext context;
```

Comment:
If it's properties class, please don't autowire the `ApplicationContext` here, make it a pure properties class.